bazel/linting/deps.bzl

```diff
@@ -0,0 +1,13 @@
+load("//bazel/utils:binary.bzl", "download_binary")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def lint_deps_init():
+    http_archive(
```

Comment:
wrap this in `maybe`? similar to other _init rule (except maybe did not exist back then, so they do it the old manual way).

https://docs.bazel.build/versions/1.1.0/repo/utils.html#maybe